README.md

```diff
@@ -72,6 +121,9 @@ This covers a subset of configuration options that are most likely to be changed
 * `runtime.logging.level` - verbosity of the logs. Default - `warning`. Recommended to increase to `debug` when troubleshooting.
 * `runtime.use_exporters` **(work in progress)** - enable other exporters. More on this in [Exporter API](#exporter-api). Default: `false`.
 * `runtime.watchers` - list of enabled watchers (collectors). More on this in [Watchers](#watchers).
+
+Agent exposes the following configuration parameters to tune its runtime:
+
```

Comment:
Seems like something is missing after this sentence.

Reply:
Addressed in 03e5550.